src/components/artistas/ArtistaForm.jsx

```diff
@@ -245,6 +250,58 @@ class ArtistaForm extends Component {
     }
   }
 
+  handleVimeoChange(id) {
+
+    if(id !== undefined && id !== null && id !== '') { // If id has a value
+
+      // Call to get the video data
+      const videoData = getVimeoData(id, response => {
+
+        // Inital empty sources
+        let sources = {};
+
+        if (response.error) { // Something wetn wrong :(
+          toastr.warning('Error', 'El ID de Vimeo no es correcto', ToastrOptionsError);
```

Comment:
This happens if anything goes wrong. The error returned does actually (in eng) tell you the issue. In my case I don't have an access token rather than the ID being wrong) Not sure if warning the user possibly incorrectly is bad UX or not?